src/homepage/index.html

```diff
@@ -225,6 +231,7 @@ <h2 class="big-h2"><span class="fa fa-wrench"></span> <span>James is a <b>living
       <section>
         <h2>Connect</h2>
         <ul class="icons">
+          <li><a href="https://gitter.im/apache/james-project" class="icon fa-wechat alt"><span class="label">Gitter</span></a></li>
```

Comment:
what about trying to have a similar ordering of media in footer and in community ?